nameko/testing/pytest.py

```diff
@@ -23,14 +23,29 @@ def pytest_addoption(parser):
         help=("The logging-level for the test run."))
 
     parser.addoption(
-        "--amqp-uri", action="store", dest='AMQP_URI',
-        default='amqp://guest:guest@localhost:5672/:random:',
-        help=("The AMQP-URI to connect to rabbit with."))
+        "--rabbit-host", action="store", dest='RABBIT_HOST',
```

Comment:
Because it's not possible to specify an `AMQP_URI` _without_ a vhost. Even `amqp://localhost` implies the default vhost.